File: LeetCode/0max-consecutive-ones/0max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max_cnt = 0, count = 0;
        for (int num : nums) {
            if (num == 1) max_cnt = Math.max(max_cnt, ++count);
            else count = 0;
        }
        return max_cnt;
    }
}

File: 0max-consecutive-ones/0max-consecutive-ones.c
int findMaxConsecutiveOnes(int* nums, int numsSize) {
    int max_cnt = 0;
    int cnt = 0;

    for (size_t i = 0; i < numsSize; i++) {
        if (nums[i] == 1) cnt++;
        else {
            if (max_cnt < cnt) max_cnt = cnt; 
            cnt = 0;
        }
    }
    
    if (max_cnt < cnt) max_cnt = cnt;
    
    return max_cnt; 
}